### PR TITLE
docs(site): updated directions for Google Cloud access

### PR DESCRIPTION
**Pull request by basldfalksjdf**
Updated directions for Google Cloud access. You don't need Google Cloud SDK.

### CODE REVIEW

File: site/content/docs/Repositories/_index.md [hunk 1]
```diff
@@ -52,6 +54,29 @@ gs://kopia-test-123/n417ffc2adc8dbe93f1814eda3ba8a07c
...
+***Method #3: Enabling AWS S3 Interoperability in Google Cloud Storage
+
+1. Create a storage bucket in [Google Cloud Console](https://console.cloud.google.com/storage/)
+2. Go to [Settings and then Interoperability](https://console.cloud.google.com/storage/settings;tab=interoperability) in your Google Cloud Storage account
+3. Enable your project under "Default project for interoperable access" and generate access keys for this project -- you will generate both access key and secret key, just like if you were using AWS S3
+
+After these preparations we can create Kopia repository using the s3 function (assuming bucket named `kopia-test-123`):
+
+```shell
+$ kopia repository create s3 --bucket="kopia-test-123" --access-key="access/key/here" --secret-access-key="secret/key/here"
```
Comment (jkowalski) [inline site/content/docs/Repositories/_index.md:77]: I think you need to pass endpoint and region here...

File: site/content/docs/Repositories/_index.md [hunk 2]
```diff
@@ -60,6 +85,18 @@ To connect to a repository that already exists, simply use `kopia repository con
...
+or 
+
+```shell
+$ kopia repository connect gcs --credentials-file="/path/to/your/credentials/file.json" --bucket kopia-test-123
+```
+
+or 
+
+```shell
+$ kopia repository connect s3 --bucket="kopia-test-123" --access-key="access/key/here" --secret-access-key="secret/key/here"
```
Comment (jkowalski) [inline site/content/docs/Repositories/_index.md:97]: same here.

File: site/content/docs/Repositories/_index.md (hunk 1 above)
Comment (basldfalksjdf) [reply site/content/docs/Repositories/_index.md]: You are right, sorry forgot that when copy and pasting. Updated the pull, not too familiar with github, so I hope you can see it. If not, this needs to be added to the s3 command when using s3 for gcs: `--endpoint="storage.googleapis.com"`

File: site/content/docs/Repositories/_index.md (hunk 2 above)
Comment (basldfalksjdf) [reply site/content/docs/Repositories/_index.md]: Same comment as above

**Pull request merged**